### DISCUSSION TITLE
How to separate mailboxes on storage fs?

### DISCUSSION THREAD

**Discussion by sabinich** [emersion/maddy#578] [General]
Hello!
I've read documentation but don't understand one thing: how to separate mailboxes on storage fs?
When I've been checking two email-accounts on different domains, every mail stored into one default `message` folder.

**Comment by Chris-Scot** [comment]
I can do this
sql local_mailboxes {
  imap_filter command /var/lib/maddy/filters/{account_name}.sh
}
Or this.
  imap_filter command /etc/maddy/sieve.sh {account_name}

So why can't I do this?
storage.imapsql {
    driver sqlite3
    dsn imapsql.db
    msg_store fs messages/{account_name}/
}
